android/src/main/java/com/jaumard/smsautofill/AppSignatureHelper.java

```diff
@@ -73,7 +74,7 @@ private static String hash(String packageName, String signature) {
         String appInfo = packageName + " " + signature;
         try {
             MessageDigest messageDigest = MessageDigest.getInstance(HASH_TYPE);
-            messageDigest.update(appInfo.getBytes(StandardCharsets.UTF_8));
+            messageDigest.update(appInfo.getBytes(Charset.forName("UTF-8")));
```

Comment:
why this change ? I rather not depend on a string if a constant exist

Reply:
`StandardCharsets.UTF_8` is not compatible with older android versions

Reply:
Ho ok :) LGTM then ;) Thanks a lot !